packages/use-subscription/src/__tests__/useSubscription-test.js

```diff
@@ -595,7 +597,7 @@ describe('useSubscription', () => {
       React.startTransition(() => {
         mutate('C');
       });
-      await waitFor(['render:first:C', 'render:second:C']);
```

Comment:
I'm not sure about the intended semantics of `waitFor` but I'm confused it doesn't wait just as long or longer than `waitForPaint`.